Source/Scene/Scene.js

```diff
@@ -3311,5 +3311,27 @@ define([
         return destroyObject(this);
     };
 
+     /**
+     * Transforms a position in WGS84 coordinates to window coordinates.  This is commonly used to place an
+     * HTML element at the same screen position as an object in the scene.
+     *
+     * @param {Cartesian3} position The position in WGS84 (world) coordinates.
+     * @param {Cartesian2} [result] An optional object to return the input position transformed to window coordinates.
+     * @returns {Cartesian2} The modified result parameter or a new Cartesian2 instance if one was not provided.  This may be <code>undefined</code> if the input position is near the center of the ellipsoid.
+     *
+     * @example
+     * // Output the window position of longitude/latitude (0, 0) every time the mouse moves.
+     * var scene = widget.scene;
+     * var ellipsoid = scene.globe.ellipsoid;
+     * var position = Cesium.Cartesian3.fromDegrees(0.0, 0.0);
+     * var handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+     * handler.setInputAction(function(movement) {
+     *     console.log(scene.wgs84ToWindow(position));
+     * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+     */
+    Scene.prototype.wgs84ToWindow = function(position, result) {
```

Comment:
My initial suggestion was going to be to keep `wgs84ToWindowCoordinates` as the name, but looking into it some more, I think having `wgs84` in the name at all is actually incorrect.  I don't believe this always uses WGS84 and instead uses the Scene's ellipsoid (which may be something non-wgs84).  @bagnell can you confirm this?

If my assumption is correct, something like `scene.computeCanvasCoordinates` may be a better choice (and obviously we would need to update the documentation to match.  Something like "The position in world coordinates" or the 
"position in the scene's ellipsoid".

Anyone else have any thoughts here?